packages/esm-service-queues-app/src/hooks/useQueueEntries.ts

```diff
@@ -0,0 +1,19 @@
+import { openmrsFetch } from '@openmrs/esm-framework';
+import { type QueueEntry, type QueueEntrySearchCriteria } from '../types';
+import useSWR from 'swr';
+
+export function useQueueEntries(searchCriteria?: QueueEntrySearchCriteria, rep: string = 'default') {
```

Comment:
I'm not 100% sure this will work in all cases.  I've seen recent cases where actually specifying "v=default" throws an error.  Which is a problem, but one we can avoid by just leaving this off altogether.  By definition, "default" will happen if no `v=...` representation is explicitly included.

Reply:
Probably a backend bug, but in my local instance and `kgh-test`, I get different responses when I omit `v=default`, and I need the fields returned by `v=default`. I'll leave this as is for now.